TheMovieDb/Networking/Endpoints.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Endpoints.swift
+//  TheMovieDb
+//
+//  Created by Juan David Torres on 03/11/21.
+//
+
+import Foundation
+
+enum API {
+  case getTrendingMovies
+  case getNowPlayingMovies
+  case getPopularMovies
+  case getTopRatedMovies
+  case getUpcomingMovies
+  case getRecommendedMovies(id: Int)
+  case getSimilarMovies(id: Int)
+  case getReviews(id: Int)
+  case searchMovies(search: String)
+}
```

Comment:
this is optional but recommended: Try to find a way to separate this, this has the risk to get bigger and bigger